packages/ui-img/babel.config.js

```diff
@@ -23,11 +23,6 @@
  */
 
 module.exports = {
-  shouldPrintComment(value) {
```

Comment:
There is still a @ts-expect-error in the ui-img folder ( in ImgLocator.ts). Shouldn't we wait with the removal of shouldprintcomment?

Reply:
I put it back as we discussed!